README.md

```diff
@@ -127,7 +129,7 @@ struct FootballMatchApp: Widget {
 
   ```dart
   final _liveActivitiesPlugin = LiveActivities();
-  _liveActivitiesPlugin.init(appGroupId: "YOUR_CREATED_APP_ID");
+  _liveActivitiesPlugin.init(appGroupId: "YOUR_GROUP_ID");
```

Comment:
Previously this id was referred to both as "YOUR_CREATED_APP_I" and as "YOUR_GROUP_ID". For consistency and clarity, I've renamed all instances to the same name.